tor/core/user_interaction.py

```diff
@@ -166,17 +172,30 @@ def process_done(post, config, override=False, alt_text_trigger=False):
                 logging.info('Moderator override starting!')
             # noinspection PyUnresolvedReferences
             try:
-                if alt_text_trigger:
+
+                redis_result = config.redis.hget(
```

Comment:
hget / hset does not allow the same person to claim more than one at a time, but there is no language or error that reflects that.

Reply:
I didn't think of that, I'll see if I can fix that this weekend (or maybe sooner). I guess I should flip the key and field?

Reply:
If we flip the hash table key and value so it's keyed by the post id, I feel like that should resolve it. Since the author name is immutable, I don't foresee any issues with user configuration making them lose their claims either.

Reply:
:+1: